config-connector/solutions/iam/helm/folder-iam/README.md

```diff
@@ -0,0 +1,87 @@
+# Folder IAM
+
+==================================================
+
+## NAME
+
+  folder-iam
+
+## SYNOPSIS
+
+  Config Connector compatible YAML files to grant a specific member a role (default to `roles/resourcemanager.folderEditor`) to an existing folder.
+
+## CONSUMPTION
+
+  1. Clone GoogleCloudPlatform/cloud-foundation-toolkit repository:
+  
+      ```bash
+      git clone https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit.git
+      ```
+
+  1. Go to the folder-iam folder:
+
+      ```bash
+      cd cloud-foundation-toolkit/config-connector/solutions/iam/helm/folder-iam
+      ```
+
+## REQUIREMENTS
+
+1. GKE Cluster with Config Connector and [Workload Identity](https://cloud.google.com/kubernetes-engine/docs/how-to/workload-identity#enable_workload_identity_on_a_new_cluster).
+
+1. A working Config Connector cluster using the cnrm-system service account with _at least_ the following role in the desired folder:
```

Comment:
I would tweak the wording slightly to something like "_minimally_ the permissions given by the following role on the desired folder", or something else that replaces "at least" with "minimally" or "minimum", because I think "at least" implies that users **must** have the indicated role, while we want to indicate a stronger one will do.

Reply:
I got your point. Rephrased the sentence.